packages/shared-ux/router/impl/services.ts

```diff
@@ -63,12 +63,12 @@ const defaultContextValue = {
   services: {},
 };
 
-export const sharedUXContext =
+export const SharedUXContext =
```

Comment:
Can we please rename this into `SharedUXRouterContext`

Reply:
Changed in [c56ab90](https://github.com/elastic/kibana/pull/204547/commits/c56ab90169db20ce341fd6bf9f12442e74a6c3b3)